'屎就放心中'

westee opened the pull request.

blindpirate commented on src/main/java/com/github/hcsp/maven/Version.java:
```diff
@@ -13,5 +16,40 @@
+        int length = Math.max(versionArr1.size(), versionArr2.size());
+
+        if (versionArr1.size() < length) {
+            System.out.println(versionArr1.size());
```
生产代码中你还打算放这么多`System.out.println`语句么。。。

westee commented:
老师说的对。学生记下了 904885508 邮箱：904885508@qq.com 签名由 网易邮箱大师 定制 在2019年07月25日 13:47，Bo Zhang 写道： @blindpirate commented on this pull request. In src/main/java/com/github/hcsp/maven/Version.java: > @@ -13,5 +16,40 @@

      * @param version2 传入的版本字符串2，支持x/x.y/x.y.z，你可以假定传入的字符串一定是合法的语义化版本

      * @return -1/0/1 当version1 小于/等于/大于 version2时

      */

-    public static int compare(String version1, String version2) {}

+    public static int compare(String version1, String version2) {

+        ArrayList<String> versionArr1 = new ArrayList<>(Arrays.asList(version1.split("\\.")));

+        ArrayList<String> versionArr2 = new ArrayList<>(Arrays.asList(version2.split("\\.")));

+

+        int length = Math.max(versionArr1.size(), versionArr2.size());

+

+        if (versionArr1.size() < length) {

+            System.out.println(versionArr1.size());

 生产代码中你还打算放这么多System.out.println语句么。。。 — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or mute the thread.
